dev_docs/mcp_design.md

```
@@ -200,6 +230,14 @@ pub struct StartResultDto {
   ]
 }
 ```
+**Bounded wait option**
+```json
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)